static/images/diagrams/image_workflow.txt

```diff
@@ -0,0 +1,42 @@
+# Use http://sequencediagram.org/ to generate image
+
+title Image-based provisioning with Puppet
+
+participant User
+participant "Provisioned Host" as Host
+participant "Foreman" as Foreman
+participant "Foreman Proxy" as Proxy
+participant "Hypervisor" as HVM
+participant DHCP
+participant DNS
+participant "Puppet master" as Puppet
+
+linear
+autonumber
+
+note over Foreman:has an Image with credentials defined
+User->Foreman:clicks on New Host
+User->Foreman:selects virtualization resource and submits
+Foreman->HVM:creates new VM
+HCM->Foreman:reports MAC address
+Foreman->Proxy:requests free IP address (using the MAC)
```

Comment:
This happens without the MAC, just on the New Host form (between line 18/19) before the host is submitted and saved.